.github/CHANGELOG.md

```diff
@@ -2,6 +2,28 @@
 
 <h3>New features since last release</h3>
 
+* VQE problems can now intuitively been set up by passing the Hamiltonian 
+  as an observable. [(#1474)](https://github.com/PennyLaneAI/pennylane/pull/1474)
+
+  ``` python
+  dev = qml.device("default.qubit", wires=2)
+  H = qml.Hamiltonian([1., 2., 3.],  [qml.PauliZ(0), qml.PauliY(0), qml.PauliZ(1)])
+  w = qml.init.strong_ent_layers_uniform(1, 2, seed=1967)
+  
+  @qml.qnode(dev)
+  def circuit(w):
+      qml.templates.StronglyEntanglingLayers(w, wires=range(2))
+      return qml.expval(H)
```

Comment:
Since nothing needed to be changed in the `measure.py` file, does that mean that `sample(H)` and `var(H)` also automatically works?

Reply:
Ha! I thought so (and manually excluded these cases because the `hamiltonian_expand` logic doesn't reproduce the measurement type), but it turns out `var` can only be defined for observables, which the Hamiltonian class is not (at this stage).

Reply:
Ah, that makes sense! This might be a one-line change in `pennylane/measure.py::var`.

Reply:
Thinking about it more, I actually don't know how `sample` is supposed to work. The postprocessing_fn would have to turn samples of Pauli observables into samples of the Hamiltonian. Is this obvious?

Same for variance, which is not a linear operator that can just be recombined like expectations, no?

Reply:
But at the moment we do not support computations of Hamiltonian samples or variance anyways. I think this needs to become a future ADR, it could be bigger.

Reply:
Oh this is a very good point - for variances, we would need the covariances between the Pauli groups in order to get `var(H)` 🤔 

For samples though, if we treat them as one-shot expectation values, would the same linearity as the expval hold?

Reply:
> But at the moment we do not support computations of Hamiltonian samples or variance anyways. I think this needs to become a future ADR, it could be bigger.

True. It's a nice thing to have, but maybe there is a more flexible framework for this. Actually, now that I think about it, there was a user request this week for efficient Hamiltonian sampling in `ExpvalCost` (https://discuss.pennylane.ai/t/differentiable-circuit-with-sampling/1190/19)

Reply:
Outch :/ 

I added an exploration ticket to the epic, but it *is* a new feature request, so strictly speaking out of scope. Let's see what we can do!